docs/introduction.rst

```diff
@@ -618,7 +618,8 @@ method creates a kind of `defaultdict` like that in the `collections` module.
 
     >>> class DefaultSortedDict(SortedDict):
     ...     def __missing__(self, key):
-    ...         return 0
+    ...         self[key] = 0
+    ...         return self[key]
```

Comment:
```suggestion
    ...         return 0
```

Reply:
Why not simply return 0?

Reply:
O, maybe because in the list case you want the same reference to the new list returned.

I think a better pattern would be to assign the new thing to a temporary variable, assign the temporary variable into the mapping, and then return the temporary variable.